invokeai/frontend/web/src/features/parameters/components/Upscale/ParamTileSize.tsx

```diff
@@ -0,0 +1,53 @@
+import { CompositeNumberInput, CompositeSlider, FormControl, FormLabel } from '@invoke-ai/ui-library';
+import { useAppDispatch, useAppSelector } from 'app/store/storeHooks';
+import { selectTileSize, tileSizeChanged } from 'features/parameters/store/upscaleSlice';
+import { useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+
+const initial = 1024;
+const sliderMin = 512;
+const sliderMax = 1536;
+const numberInputMin = 512;
+const numberInputMax = 1536;
+const coarseStep = 64;
+const fineStep = 32;
```

Comment:
Are there model arch constraints to the tile size? Maybe we should just leave this at 64px.

Reply:
I don't _think_ so (think its multiple of 8), but can keep at 64 without much fuss.